enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -1002,7 +1002,23 @@ components by the Ingress Operator.
+| 1b  | As a user I want to create, modify, and delete a  GatewayClass, Gateway, HTTPRoute, and ReferenceGrant.                                              | When an instance is created, the status shows Ready. When an instance is deleted, all supporting objects are also deleted.                                                                                                                  |
+| 2   | As a user I want to connect a backend application  to an HTTPRoute in the same namespace.                                                            | A user can connect a backend application to an HTTPRoute in  the same namespace.  The HTTPRoute status is Ready.                                                                                                                            |
+| 3   | As a user I want to be able to access the application behind an HTTPRoute via curl to an HTTP or  edge-terminated HTTPS endpoint.                    | A user can access the application behind an HTTPRoute via curl to an HTTP or edge-terminated HTTPS endpoint.                                                                                                                                |
+| 4   | As a user I want to be able to create a ReferenceGrant to target a service in one namespace from an HTTPRoute in another namespace.                  | A user can create a ReferenceGrant to target a service in one namespace from an HTTPRoute in another namespace.  The HTTPRoute status is Ready.  The user can access the application via curl to an HTTP or edge-terminated HTTPS endpoint. |
+| 5   | As a user, I want to be able to view the GatewayClass, Gateway, and HTTPRoute CRDs.                                                                  | Gatewayapi_controller creates the GatewayClass, Gateway, HTTPRoute, and ReferenceGrant CRDs when the GatewayAPI feature gate is enabled.                                                                                                    |
+| 6   | As a user, I want to be able to view the status of OSSM and SMCP.                                                                                    | Gatewayclass_controller installs OSSM and a ServiceMeshControlPlane, if they don’t already exist, after a GatewayClass owned by OpenShift is created.  The status is Ready for both and logs show no error.                                 |
+| 7   | As a user I want to find a wildcard DNS record ready after I add a Listener to  a Gateway.                                                           | When a user creates a Gateway with a Listener, a wildcard DNS record is created.  If the Listener is modified or deleted, the DNS record is updated or removed.                                                                             |
```

Comment:
Sorry, I know we want to get this in asap, but one possibly misleading detail: the DNS Record does not have to be a wildcard. It's only a wildcard record (e.g. `*.example.com`) if the Gateway's Listener's `.hostname` is a wildcard. If the Listener's `.hostname` is `app.example.com`, the DNS record is not a wildcard. I made a [slide](https://docs.google.com/presentation/d/1wvm5KK3nAbpr_ZEpNQhK3Aj7vVIwiouuy-cZ7xRBBxY/edit#slide=id.g23da83c2f62_0_11) on this in the demo for visualization. I'd suggest to drop it to avoid confusion:
```suggestion
| 7   | As a user, I want to find a DNS record ready after I add a Listener to a Gateway.                                                           | When a user creates a Gateway with a Listener, a DNS record is created.  If the Listener is modified or deleted, the DNS record is updated or removed.                                                                             |
```

Reply:
Just a reminder on this

Reply:
Thanks @gcs278 - I made the suggested changes and added a few commas.  PTAL.